build.sbt

```diff
@@ -5,10 +5,9 @@ val customScalaJSVersion = Option(System.getenv("SCALAJS_VERSION"))
 val scalaJSVersion = customScalaJSVersion.getOrElse("1.0.0")
 val scalaNativeVersion = "0.4.0-M2"
 def scala213 = "2.13.1"
-def scala212 = "2.12.10"
+def scala212 = "2.12.11"
 def scala211 = "2.11.12"
 def dotty = "0.23.0-RC1"
-def scalameta = "4.3.0"
```

Comment:
again, not strictly related, but this was unused

Reply:
👍